test/app-tap/module_api.c

```diff
@@ -2335,6 +2335,80 @@ test_tuple_validate_formatted(lua_State *L)
 	return 1;
 }
 
+static int
+session_check_admin(lua_State *L)
```

Comment:
Why is it called check admin, if in this function nothing is related to admin?

Reply:
This part of `modul_api.c` copied from https://github.com/tarantool/tarantool/blob/master/test/box-tap/session.test.lua#L21-L28. Maybe, another name is more appropriate.